templates/terraform/custom_flatten/cloudscheduler_job_appenginerouting.go.erb

```diff
@@ -0,0 +1,26 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2019 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+// An `appEngineRouting` filed of CloudScheduler API only contains output-only `host` field,
+// so we set config values rather than api response to state.
+// https://cloud.google.com/scheduler/docs/reference/rest/v1/projects.locations.jobs#AppEngineRouting
+func flatten<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d *schema.ResourceData) interface{} {
+	if v == nil {
+		return nil
+	}
+	if targets := d.Get("app_engine_http_target").([]interface{}); len(targets) > 0 {
+		return targets[0].(map[string]interface{})["app_engine_routing"]
+	}
+	return nil
```

Comment:
Another choice is doing like this; I don't know which is better...

```go
return []interface{}{map[string]interface{}{
	"service":  d.Get("app_engine_http_target.0.app_engine_routing.0.service"),
	"version":  d.Get("app_engine_http_target.0.app_engine_routing.0.version"),
	"instance": d.Get("app_engine_http_target.0.app_engine_routing.0.instance"),
}}
```

Reply:
Looks better, I adopted your suggestion! ea6171a285ec2865c8010bc98af643c1f9cd9eff